lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -137,17 +140,17 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                   return success();
                 });
   patterns.onOp("Mul", 7,
-		[](OpBinder binder, ConversionPatternRewriter &rewriter) {
-		  Torch::ValueTensorType resultType;
+                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+                  Torch::ValueTensorType resultType;
                   Value lhs, rhs;
                   if (binder.tensorOperands(lhs, rhs) ||
                       binder.tensorResultType(resultType)) {
                     return failure();
                   }
                   rewriter.replaceOpWithNewOp<Torch::AtenMulTensorOp>(
-		    binder.op, resultType, lhs, rhs);
```

Comment:
thanks. The format changes are the results of clang-format, which is recommended in the torch-mlir build instructions. Next time, I'll just try clang-format on my own changes.

Reply:
Yeah I've seen this a few times, but seems like something got messed up with torch-mlir's clang format somewhere along the way and we need a reset. I don't have a great idea of how to avoid this so I guess it's fine for now, but this really does mess up the blame log.